README.md

```diff
@@ -21,13 +21,14 @@ This project is considered feature complete for use as a soldering iron, _so ple
 _This firmware does **NOT** support the USB port while running for changing settings. This is done through the onscreen menu only. Logos are edited using the tool or python script and uploaded in DFU mode._
 
 
-|  Device 	| DC Support| QC Support | PD Support | EPR Support | Battery | Recommended |
-|--------:	|:---------:|:----------:|:----------:|:----------:|:--------:|:-----------:|
-|   MHP30 	|     ❌    |    ❌     |     ✔️    |     ❌     |    ❌   |      ✔️     |
-|  Pinecil 	|     ✔️    |    ✔️     |     ✔️    |     ✔️*    |    ✔️   |      ✔️     |
-|   TS100 	|     ✔️    |    ❌     |     ❌    |     ❌     |    ✔️   |      ❌     |
-|   TS80    |     ❌    |    ✔️     |     ❌    |     ❌     |    ✔️   |      ❌     |
-|   TS80P 	|     ❌    |    ✔️     |     ✔️    |     ❌     |    ✔️   |      ✔️     |
+|    Device | DC | QC | PD | EPR | BLE | Battery | Recommended |
+|----------:|:--:|:--:|:--:|:---:|:---:|:-------:|:-----------:|
```

Comment:
I prefer to keep these in alphabetical order if possible

Reply:
As you wish! 😃